### PR TITLE
Adding Users app

### PR DESCRIPTION
**Pull request by SaarOrus**
This PR adds a Users app, which includes the built-in django user and additional fields such as:  `phone_number` field, `image_url` field that stores a URL for the user's profile picture and 'date_of_birth' field.

Saar orus <saar4848@gmail.com>

Resolve : #34

### CODE REVIEW

**Review by rotem2022 — APPROVED** [review]

**Review by Ofekmn — DISMISSED** [review]

File: users/models.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    date_of_birth = models.DateTimeField()
+    phone_number = models.CharField(max_length=15)
+    image_url = models.CharField(max_length=40)
```
Comment (ygalblum) [inline users/models.py:9]: 40 might be to short in some cases. No need to fail on this. From a quick look, I found the Django has a URLField model: https://docs.djangoproject.com/en/4.2/ref/models/fields/#urlfield

**Review by ygalblum — CHANGES_REQUESTED** [review]

File: users/models.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+from django.contrib.auth.models import User
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    date_of_birth = models.DateTimeField()
+    phone_number = models.CharField(max_length=15)
+    image_url = models.URLField(max_length=200, blank=True)
+
+    def __str__(self):
```
Comment (ormergi) [inline users/models.py:11]: Why do we need this method?

If it is not used anywhere, you can remove it for now and add it when needed.

**Review by ormergi — CHANGES_REQUESTED** [review]
Hi Saar, please squash intermediate commits and fix the commit message to have a proper description (elaborate what this change does and why you need it) and a sign-off.
Also fix the PR description to explain what this PR does and why you need it, referencing to an issue is nice but its no replacement for the description.

File: users/models.py (hunk 2 above)
Comment (ygalblum) [reply users/models.py:11]: I've seen it in other models as well, I think it's auto-generated, no? `__str__` is the string representation of an instance. So, you won't see any direct calls to it

Comment (ormergi) [reply users/models.py:11]: From what I remember it's not auto-generated, @SaarOrus is its method auto-generated?

Comment (SaarOrus) [reply users/models.py:11]: @ormergi No, the __str__ method is not auto-generated.

**Review by ormergi — APPROVED** [review]
Thanks for the changes

**Review by ygalblum — APPROVED** [review]

**Pull request merged**